src/web/components/ApiKeyManagement/KeyTable.stories.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable camelcase */
+import type { Meta, StoryObj } from '@storybook/react';
+
+import KeyTable from './KeyTable';
+
+const meta: Meta<typeof KeyTable> = {
+  component: KeyTable,
+  title: 'ApiManagement/API key table',
+};
+export default meta;
+
+type Story = StoryObj<typeof KeyTable>;
+
+export const ManyKeys: Story = {
+  args: {
+    apiKeys: [
+      {
+        contact: 'ApiKey1',
+        name: 'ApiKey1',
+        created: 1702860516,
+        key_id: 'PTFD_12',
+        disabled: false,
+        roles: [],
+        service_id: 0,
+      },
+      {
+        contact: 'ApiKey2',
+        name: 'ApiKey2',
+        created: 1701210253,
+        key_id: 'FDSL,089',
+        disabled: false,
+        roles: [{ id: 1, roleName: 'Role1', externalName: 'Role 1' }],
```

Comment:
As above - it'll be a bit easier to think about these and get helpful feedback if the roles match the actual roles we use.